packages/server/src/main/java/software/uncharted/terarium/hmiserver/models/simulationservice/interventions/Intervention.java

```diff
@@ -32,4 +34,36 @@ public Intervention clone() {
 		}
 		return intervention;
 	}
+
+	// Check for no negative timesteps
+	// Check for no duplicate time + appliedTo pairs in static interventions
+	public Boolean validateIntervention() throws Exception {
```

Comment:
I think that these `validate` methods on `Intervention` and `InterventionPolicy` should possibly exist in `InterventionController` or `InterventionService` since it doesn't really have anything to do with the model class.

Also, this method is only returning `true` or throwing an exception... it can never be `false` not valid.